dockerplugin.py

```diff
@@ -90,22 +90,34 @@ def read_blkio_stats(container, dimensions, stats, t):
         # numbers). We need to group and report the stats of each block
         # device independently.
         blkio_stats = {}
+        blkio_major_stats = {}
+        blkio_minor_stats = {}
+
         for value in values:
+
             k = '{key}-{major}-{minor}'.format(key=key,
```

Comment:
Well the point of using the dimensions is that you no longer need to overload the metric name with that information, isn't it?